samples/servicecontrol/monitoring3rdparty/sample.md

```diff
@@ -10,23 +10,19 @@ redirects:
  - samples/custom-checks/monitoring3rdparty
 ---
 
-
-## Code walk-through
-
 When integrating with third-party systems, often messages fail when those systems are down. The Particular Platform has extensibility to enable the monitoring of third-party systems which are exposed as HTTP endpoints. This sample shows how to achieve this with custom checks.
 
+downloadbutton
```

Comment:
Based on what convention do we use the downloadbutton? I thought we only add it to samples that we consider worth highlighting?